src/OrbitGl/ViewportTest.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "Viewport.h"
+
+namespace orbit_gl {
+
+TEST(Viewport, ResizingAndDirty) {
+  Viewport viewport(100, 200);
+  EXPECT_TRUE(viewport.IsDirty());
+  viewport.ClearDirtyFlag();
+
+  // Test initial values
```

Comment:
Nit: Consider adding punctation to the comments here.

Reply:
Done